units/systemd-user.service.in

```diff
@@ -9,6 +9,8 @@ Environment="XDG_DATA_HOME=%h/.local/share"
 ExecStart = BINPATH`'DAEMONNAME -vv
 ExecReload = /bin/kill -HUP $MAINPID
 CPUQuota=1%
+Restart=Always
```

Comment:
I would use here `on-failure` not `Always` so we restrict restarting only on issues.

Reply:
Fixed in commit ab375f26f13c23c6f3080b0ca6b13e4bc1fb339e. :+1: